### PR TITLE
feature/docker-compose-rework

### PR DESCRIPTION
**Pull request by Andreichenko**
added

### CODE REVIEW

**Comment by Andreichenko** [comment]
please check @freideveloper

File: docker-compose.yml [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+    restart: always
+    container_name: "salt-master"
+    hostname: "devops-salt-master"
+    volumes:
+     - ./salt/:/srv/salt
+     - ./pillar/:/srv/pillar
+    ports:
+      - '22033:22'
+    depends_on:
+      - "salt-minion-2"
```
Comment (freideveloper) [inline docker-compose.yml]: not necessary because we have to up master

File: docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+  client2:
+    build: ./salt-minion
+    privileged: true
+    restart: always
+    container_name: "salt-minion-2"
+    hostname: "salt-minion-2"
+    ports:
+      - '22035:22'
+    depends_on:
+      - "salt-minion-1"
```
Comment (freideveloper) [inline docker-compose.yml]: the same

**Review by freideveloper — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -37,11 +36,10 @@ services:
     hostname: "salt-minion-2"
     ports:
       - '22035:22'
-    depends_on:
-      - "salt-minion-1"
```
Comment (freideveloper) [inline docker-compose.yml]: okay

File: docker-compose.yml [hunk 4]
```diff
@@ -12,8 +12,7 @@ services:
      - ./pillar/:/srv/pillar
     ports:
       - '22033:22'
-    depends_on:
-      - "salt-minion-2"
```
Comment (freideveloper) [inline docker-compose.yml]: Okay

**Review by freideveloper — APPROVED** [review]

**Pull request merged**